src/inputs/BlockEditor-slate/preview/List.js

```
@@ -0,0 +1,28 @@
+import React, {PropTypes} from 'react'
+import styles from '../styles/List.css'
+
+function List(props) {
+  const listStyle = props.listStyle || props.children[0].props.parent.data.get('listStyle')
+  const classNames = [styles.root]
+  if (props.preview) {
+    classNames.push(styles.preview)
+  }
+  if (listStyle === 'bullet') {
+    classNames.push(styles.bullet)
+    return <ul className={classNames.join(' ')}>{props.children}</ul>
+  }
+  if (listStyle === 'roman') {
+    classNames.push(styles.roman)
+    return <ol className={classNames.join(' ')}>{props.children}</ol>
+  }
+  classNames.push(styles.number)
+  return <ol className={classNames.join(' ')}>{props.children}</ol>
```

I'd prefer it to fail hard if I have provided an invalid listStyle.